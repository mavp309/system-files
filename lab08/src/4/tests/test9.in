-2.34 -0.88 3.7 4.9
