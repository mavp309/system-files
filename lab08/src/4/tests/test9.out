0.1100 0.9700
